src/Card/Card.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx jsx */
+import * as colors from "../colors";
+import { base } from "../typography";
+import { jsx } from "@emotion/core";
+import React from "react";
+import PropTypes from "prop-types";
+
+interface Props {
+  children?: React.ReactNode;
+  title?: React.ReactNode;
+  className?: string;
+  description?: React.ReactNode;
+  titleChildren?: React.ReactNode;
+  forceNoChildPadding?: boolean;
+  size?: "standard" | "large";
+}
+
+export const Card: React.FunctionComponent<Props> = ({
+  children,
+  forceNoChildPadding,
+  title,
+  titleChildren,
+  description,
+  size,
+  ...otherProps
+}) => (
+  <div
+    {...otherProps}
+    css={{
+      backgroundColor: colors.white,
+      color: colors.black.base,
+      boxShadow: `0 4px 8px 0 rgba(0, 0, 0, .04)`,
+      borderRadius: `0.5rem`,
+      borderWidth: 1,
+      borderColor: colors.silver.dark,
+      paddingLeft: 24,
+      paddingRight: 24,
+      margin: "2rem",
+      paddingTop: size === "large" ? 16 : 28,
+      paddingBottom: size === "large" ? 16 : 28,
+    }}
+  >
+    <div css={{ display: "flex" }}>
+      <div css={{ flex: "1 1 0%" }}>
+        <div css={{ display: "block" }}>
+          {title && (
+            <div
+              css={{
+                display: "flex",
+                color: colors.black.base,
+                ...(size === "large" ? base.xxlarge : base.large),
+              }}
+            >
+              <span
+                css={{
+                  fontWeight: 600,
```

Comment:
Does this size conform to something in our typography? If not, then we should talk to @adamatorres about changing it.

Reply:
this corresponds to the base typography, which doesn't have a set fontWeight so we override it to make it semibold. The same thing is done [here in button](https://github.com/apollographql/space-kit/pull/26/files#diff-9b4a3ce6a1b69789a02162483b7f8141R146)

Reply:
Why are you also overriding the `lineHeight`?

Reply:
As per [Slack convo](https://meteor.slack.com/archives/CJM405HJB/p1565054671017800) - we're going to export additional typographies that include the font weight option.  IMO we can do that in a separate PR.